updated_main.py

```diff
@@ -3,57 +3,86 @@
 import bs4
 import time
 import json
+import os
+from email.message import EmailMessage
+import ssl
+import smtplib
 
 # Running a for loop to get the latest price of the GBP
 for i in range(100):
 
     # Now extracting the value of British Pound in Indian Rupees
-    currency = "convert 1 pound to rs"
+    def Getting_current_price():
+        currency = "convert 1 pound to rs"
+        url = "https://google.com/search?q=" + currency
+        request_result = requests.get(url)
 
-    url = "https://google.com/search?q=" + currency
+        soup = bs4.BeautifulSoup(request_result.text, "html.parser")
+        temp = soup.find("div", class_="BNeawe iBp4i AP7Wnd").text
+        temps = temp.split(" ")
+        current_price = float(temps[0])
+    Getting_current_price()
+    
+    
+    #Semding Discord message
+    # Using webhook we send the message in the Discord server
+    def Discord_bot():
+        hook = Webhook(
+            "https://discord.com/api/webhooks/1003993992360366080/1cS8w3N2GHzW-2aAwLYZ9tpfcHHhRBiASgTIafnfJ1Q--NbC1Exat7lVVzr4mgvH3l5f"
+        )
 
-    request_result = requests.get(url)
+        # Defining the upper and lower limit to get the alert when it will be triggered
+        upper_limit = 96.50
+        lower_limit = 96.40
 
-    soup = bs4.BeautifulSoup(request_result.text, "html.parser")
+        if current_price >= upper_limit:
+            high_price_alert = (
```

Comment:
`high_price_alert` and `low_price_alert` don't need to be different names as they both go into same function `hook.send`.

Both of these variables can just be called `price_alert` and `hook.send` can be moved out of if else so that it is called only once.

Then whole if else conditions to can be extracted out into a function called `create_price_alert`.

Reply:
https://docs.microsoft.com/en-us/visualstudio/ide/reference/extract-method?view=vs-2022

Reply:
because high price and low price both if conditions give different alert message

Reply:
Since this is if else condition there is no risk of variable getting overwritten. Variable will either be set to high price or low price.

Reply:
ohhkk got it